### PR TITLE
Small changes for Vaadin

### PR DESCRIPTION
**Pull request by svenruppert**

### CODE REVIEW

File: chat-example/chat-vaadin-client/src/test/resources/shiro.ini [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[users]
```
Comment (saturnism) [inline chat-example/chat-vaadin-client/src/test/resources/shiro.ini]: @svenruppert is this file needed?

File: chat-example/chat-vaadin-client/pom.xml [hunk 2]
```diff
@@ -1,178 +1,227 @@
...
+      <activation>
+      </activation>
+
+      <repositories>
+        <repository>
+          <id>vaadin-prereleases</id>
+          <url>http://maven.vaadin.com/vaadin-prereleases</url>
+        </repository>
+        <repository>
+          <id>vaadin-snapshots</id>
```
Comment (saturnism) [inline chat-example/chat-vaadin-client/pom.xml]: do we need preleases/snapshots?

Comment (svenruppert) [reply chat-example/chat-vaadin-client/pom.xml]: oh i will clean it up

File: chat-example/chat-vaadin-client/src/test/resources/shiro.ini (hunk 1 above)
Comment (svenruppert) [reply chat-example/chat-vaadin-client/src/test/resources/shiro.ini]: same here: oh i will clean it up

**Comment by svenruppert** [comment]
ok, removed useless stuff....
